src/Microsoft.AspNetCore.CookiePolicy/CookiePolicyMiddleware.cs

```diff
@@ -5,6 +5,7 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
 using Microsoft.AspNetCore.Http.Features.Internal;
```

Comment:
See [line 30](https://github.com/aspnet/Security/blob/dougbu/feature.interfaces.561/src/Microsoft.AspNetCore.CookiePolicy/CookiePolicyMiddleware.cs#L30) in this file. The `class` references both `IResponseCookiesFeature` and the concrete `ResponseCookiesFeature`. (As I [mentioned](https://github.com/aspnet/HttpAbstractions/issues/549#issuecomment-198424718) in aspnet/HttpAbstractions#549, many of our repos use supposedly-internal classes and interfaces from HttpAbstractions.)